app/assets/javascripts/osem-schedule.js

```diff
@@ -0,0 +1,109 @@
+var conference; // Should be initialize in Schedule.initialize
+var schedule_id; // Should be initialize in Schedule.initialize
+
+var Schedule = {
+  initialize: function(conference_param, schedule_id_param) {
+    conference = conference_param;
+    schedule_id = schedule_id_param;
+  },
+  remove: function(element) {
+    var e =  $("#" + element);
+    var event_schedule_id = e.attr("event_schedule_id");
+    if(event_schedule_id != null){
+      var url = '/admin/conference/' + conference + '/event_schedule/' + event_schedule_id;
```

Comment:
Don't do that! Use instead the Rails url helper eg in a data attribute and get the whole url. This way you also don't need to initialize the schedule and conference ids.